<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="dateTimeEditPoop">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>194</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatus">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Soft</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonAddPoop">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Record specific time</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Date time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hard</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Normal</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Soft</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="buttonAddNowPoop">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDeletePoop">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
